src/styles/sidebar/components/_index.scss

```diff
@@ -37,3 +34,18 @@
 @use './ToastMessages';
 @use './TopBar';
 @use './VersionInfo';
+
+// TODO: Put these in @layer components after shared component styles have
+// been converted to Tailwind (and are contained in the components layer
+// themselves). They need to come after shared-component styles in the
+// output stylesheet.
+
+// Applies to <Link>
+.p-muted-link {
+  @apply text-color-text-light hover:text-color-text-light hover:underline;
+}
+
+// Applies to any text: strikeout effect
+.p-redacted-text {
+  @apply line-through grayscale contrast-50 text-color-text-light;
+}
```

Comment:
Think of this more as a staging area than a resting place. I'm not sure yet where these will ultimately be expressed, so I'm going to leave it loose for now. The goal here is that they are corralled and named in an easily-greppable way.

Reply:
Is the `p-` prefix a convention in the TW ecosystem or just something we're using?